### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Eastrall** [Eastrall/Ether.Network#14]
We must create a documentation for the project. Let's use the GitHub wiki pages.

**Comment by gageorsburn** [comment]
@Eastrall Not sure where I should put this, but just want to point out that for NuGet V1.1.7 the read me example is slightly wrong,

**Comment by Eastrall** [comment]
@gageorsburn You are correct, thank you for pointing this out. I've updated the README.md file
